### DISCUSSION TITLE
如何获取alist的token和用户id

### DISCUSSION THREAD

**Discussion by ztgin** [AlistGo/alist#4172] [Q&A]
如何获取alist的token
![image](https://user-images.githubusercontent.com/121748143/232280940-30d6c92d-664c-4aae-9a74-9850d30d950a.png)
谢谢大佬

**Comment by sangemajia** [comment]
获取alist的token输出到文件
curl -X POST "http://域名或者IP:5244/api/auth/login" -H 'Authorization:{{token}}' -H 'Content-Type: application/json' -d '{"userName": "用户名","password": "密码"}' | sed 's/,/\n/g' | grep "token" | sed 's/token//g'|sed 's/:/\n/g' | sed '1d' | sed 's/"//g' | sed 's/}//g' | sed 's/{//g' | sed '1d' > 123.txt
这是我查资料试出来的。那几个sed 是把官方输出信息里其他东西去掉的，输出结果最终就是token的值。然后调用出来就行了。运行一次，token变一次。curl有输出为函数的，我不会，又不是不能用🤣

**Reply by sangemajia** [reply]
这是网友整理出来的V3 API接口 里面有相关的参数，具体怎么用，谷歌百度挨个试🤣https://github.com/alist-org/alist/discussions/2501
